<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressources/ressources.qrc">
    <normaloff>:/icons/icon.ico</normaloff>:/icons/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="main_widget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="3">
     <widget class="QListWidget" name="lw_users">
      <property name="styleSheet">
       <string notr="true">QListView::item:selected {
	border: none;
    background-color: white;
    color: black;
}</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="le_room_id">
      <property name="styleSheet">
       <string notr="true">padding: 3px</string>
      </property>
      <property name="placeholderText">
       <string>Room ID (ex: 123456)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QLineEdit" name="le_chat_input">
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 3px</string>
      </property>
      <property name="placeholderText">
       <string>Enter you message here</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="le_username">
      <property name="styleSheet">
       <string notr="true">padding: 3px</string>
      </property>
      <property name="placeholderText">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="pb_send">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QListWidget" name="lw_chat_history">
      <property name="font">
       <font>
        <family>Monospace</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="3" colspan="2">
     <widget class="QPushButton" name="pb_game">
      <property name="styleSheet">
       <string notr="true">background-color: #5cdb5c</string>
      </property>
      <property name="text">
       <string>Join</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_server"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_documentation"/>
    <addaction name="actionVideo"/>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="action_open_ai">
   <property name="text">
    <string>open-ai</string>
   </property>
  </action>
  <action name="actionopen_ai">
   <property name="text">
    <string>open-ai</string>
   </property>
  </action>
  <action name="action_documentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionVideo">
   <property name="text">
    <string>Video</string>
   </property>
  </action>
  <action name="action_openai">
   <property name="text">
    <string>open-ai tokens</string>
   </property>
  </action>
  <action name="action_server">
   <property name="text">
    <string>Server tokens</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRecent_logins">
   <property name="text">
    <string>Recent logins</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../ressources/ressources.qrc"/>
 </resources>
 <connections/>
</ui>
